internal/kafka/internal/workers/kafka_mgrs/promotion/promotion_kafkas_mgr.go

```diff
@@ -0,0 +1,97 @@
+package promotion
+
+import (
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/api/dbapi"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/config"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/services"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/workers/kafka_mgrs/promotion/chain"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/workers/kafka_mgrs/promotion/internal/actions"
+	errors2 "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/errors"
```

Comment:
Let's use a more descriptive package alias than `errors2`. For example `apiErrors` or `svcErrors`, `kasfleetmanagererrors`, `kerrors`, `fleetErrors`, ...